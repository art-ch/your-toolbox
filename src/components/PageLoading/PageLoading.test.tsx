import { render } from '@testing-library/react';
import { PageLoading } from './PageLoading';

describe('PageLoading', () => {
  it('should render the page loading component', () => {
    const { container } = render(<PageLoading />);

    expect(container).toMatchSnapshot();
  });
});
