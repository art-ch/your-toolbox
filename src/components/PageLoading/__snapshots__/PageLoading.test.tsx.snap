// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageLoading should render the page loading component 1`] = `
<div>
  <div
    class="min-h-screen flex items-center justify-center"
  >
    <div
      class="animate-spin rounded-full h-12 w-12 border-t-2 border-b-2 border-gray-900 dark:border-white"
    />
  </div>
</div>
`;
